nornir/plugins/inventory/netbox.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pynetbox
```

Comment:
Don't use this, let's call directly the API using `requests`. As we are just making a few calls to the API and transforming the data to nornir format `pynetbox` doesn't really give us too much and introduces an extra dependency.